enrichment-ner/src/main/java/eu/europeana/enrichment/ner/linking/DBpediaSpotlight.java

```diff
@@ -69,7 +69,32 @@ public DBpediaResponse getDBpediaResponse(String dbpediaUrl) throws Exception {
 			if(responseStr==null) return null;
 			
 		    InputStream stream = new ByteArrayInputStream(responseStr.getBytes(StandardCharsets.UTF_8));
-		    dbpediaResp = ((DBpediaResponseHeader) unmarshaller.get().unmarshal(stream)).getResult();
+		    
```

Comment:
because the dbpedia sometimes may be not responsive dues to maintenance work, and we need to retry the request